community/open-source.md

```diff
@@ -2,6 +2,12 @@ If you have created (or know of) an open source library or product that
 uses CloudEvents, please include in the list below.
 
 Example:
+
 * [AwesomeLibrary](): descriptive text, consider additional links to docs or
   specific files, especially if the CloudEvents integration is deep in the repo
+* [microBean CloudEvents][microbean-cloudevents]: CloudEvents represented as
+  plain Java objects. Satellite projects add
+  [Jackson and CDI event integration][microbean-cloudevents-jackson-cdi].
 
+[microbean-cloudevents]: https://microbean.github.io/microbean-cloudevents/
```

Comment:
I know its a minor thing, but can you inline these hrefs instead of using bookmarks?  It makes it easier to manage because then if we need to move or remove things then everything is in one spot and people won't miss the bookmarks at the bottom of the file.

Reply:
@duglin Done